pulsar-broker/src/main/java/org/apache/pulsar/broker/service/persistent/PersistentTopic.java

```diff
@@ -661,18 +661,36 @@ void removeSubscription(String subscriptionName) {
         return delete(false);
     }
 
+    private CompletableFuture<Void> delete(boolean failIfHasSubscriptions) {
+        return delete(failIfHasSubscriptions, false);
+    }
+
+    /**
+     * Forcefully close all producers/consumers/replicators and deletes the topic. this function is used when local
+     * cluster is removed from global-namespace replication list. Because broker doesn't allow lookup if local cluster
+     * is not part of replication cluster list.
+     * 
+     * @return
+     */
+    private CompletableFuture<Void> deleteForcefully() {
```

Comment:
Can we also expose this as an option in `topics delete $MY_TOPIC --force` ?

Reply:
yes, make sense. I will create a PR for this admin api.